$NetBSD: patch-configure,v 1.8 2025/03/30 21:10:42 wiz Exp $

Patch "GNU" gettext support to not depend unnecessary on "GNU" gettext.

--- configure.orig	2018-08-30 08:24:28.000000000 +0000
+++ configure
@@ -15236,8 +15236,7 @@ extern
 #ifdef __cplusplus
 "C"
 #endif
-const char *_nl_expand_alias (const char *);
-#define __GNU_GETTEXT_SYMBOL_EXPRESSION (_nl_msg_cat_cntr + *_nl_expand_alias (""))
+#define __GNU_GETTEXT_SYMBOL_EXPRESSION (_nl_msg_cat_cntr + *bindtextdomain ("", ""))
 #else
 #define __GNU_GETTEXT_SYMBOL_EXPRESSION 0
 #endif
@@ -15273,8 +15272,7 @@ extern
 #ifdef __cplusplus
 "C"
 #endif
-const char *_nl_expand_alias (const char *);
-#define __GNU_GETTEXT_SYMBOL_EXPRESSION (_nl_msg_cat_cntr + *_nl_expand_alias (""))
+#define __GNU_GETTEXT_SYMBOL_EXPRESSION (_nl_msg_cat_cntr + *bindtextdomain ("", ""))
 #else
 #define __GNU_GETTEXT_SYMBOL_EXPRESSION 0
 #endif
